test/myboss/ParserTest.java

```diff
@@ -0,0 +1,51 @@
+package myboss;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+class ParserTest {
+
+    @Test
+    void testSplitUserCmd() {
+        String input = "todo borrow book";
+        String[] expected = {"todo", "borrow book"};
+        assertArrayEquals(expected, Parser.splitUserCmd(input));
+    }
+
+    @Test
+    void testGetTaskIndex() {
+        String[] input = {"mark", "3"};
+        int expected = 2;
+        assertEquals(expected, Parser.getTaskIndex(input));
+    }
+
+    @Test
+    void testGetRemainingUserCmd() {
+        String[] input = {"todo", "borrow book"};
+        String expected = "borrow book";
+        assertEquals(expected, Parser.getRemainingUserCmd(input));
+    }
+
+    @Test
+    void testGetRemainingUserCmd_ExitCase_EmptyString() {
```

Comment:
Do you know that there's a format to follow for test case method name as stated in Week 3 readings? E.g. `whatIsBeingTested_descriptionOfTestInputs_expectedOutcome`. I will probably name this as `getRemainingUserCmd_exitCommand_emptyString` and do the same for the other test cases.